.github/pull_request_template.md

```diff
@@ -0,0 +1,51 @@
+<!--
+    Thank you for contributing to our project!
+
+    Please do not delete this text completely, but read the text below and keep
+    items that seem relevant. If in doubt, just keep everything and add your
+    own text at the top, a reviewer will update the checklist for you.
+
+    While the checklist is intended to be filled in by the technical and scientific
+    reviewers, it is the responsibility of the author of the pull request to make
+    sure all items on it are properly implemented.
+-->
+
+## Description
+
+<!--
+    Please describe your changes here, especially focusing on why this pull request makes licenseguard better and what problem it solves.
+
+    Before you start, please read our contribution guidelines: https://github.com/NLeSC/licenseguard/blob/main/CONTRIBUTING.md
+
+    Please fill in the GitHub issue that is closed by this pull request, e.g. Closes #1903
+-->
+
+Refs: #issue_number
+
+* * *
+
+## Before you get started
+
+<!--
+    Please discuss your idea with the development team before getting started,
+    to avoid disappointment or unnecessary work later. The way to do this is
+    to open a new issue on GitHub.
+-->
+
+- [ ] [☝ Create an issue](https://github.com/NLeSC/licenseguard/issues/new/choose) to discuss what you are going to do
+
+## Checklist
+
+It is the responsibility of the author to make sure the pull request is ready to review.
```

Comment:
This sentence is a bit confusing to me. Maybe instead of this, we could have something like

"_This section to be filled in by the reviewer._"